src/doc/en/installation/index.rst

```diff
@@ -207,21 +144,3 @@ In the cloud
 - `Docker image sagemathinc/cocalc
   <https://hub.docker.com/r/sagemathinc/cocalc>`_ can be used on any system with Docker to run CoCalc locally.
 
-
-More information:
-
-.. toctree::
-   :maxdepth: 2
-
-   linux
-   binary
-   conda
-   source
-   meson
-   launching
-   troubles
```

Comment:
the warning is because the TOC tree (here) is gone, no? (the change is rather massive I haven't managed to review it, and why is the distinction between development and no-development gone anyway?)